### DISCUSSION TITLE
Cannot do a fresh install

### DISCUSSION THREAD

**Discussion by BAW277** [flybywiresim/aircraft#6139] [Help]
I am feverishly trying to do a fresh install to no avail. I emptied my %community% folder, the installer starts, downloads a full.zip, decompresses it and then quits with a copy error message. The game itself lives on my E-Drive, could this be a culprit?

Any help is greatly appreciated.

**Comment by Riggie** [comment]
If you need to get on and run the sim, just download any one of the builds from the page below to your PC,  and drop it in the Community folder
https://flybywiresim.com/a32nx/
